docker/observability/grafana/provisioning/dashboards/validator-dashboard.json

```diff
@@ -333,7 +258,7 @@
             "uid": "P1809F7CD0C75ACF3"
           },
           "exemplar": true,
-          "expr": "sum by (chain_id) (tableland_wallettracker_balance_wei{service_name=\"tableland:api\"})/1000000000",
+          "expr": "sum by (chain_id) (tableland_wallettracker_balance_wei{service_name=\"tableland:healthbot\"})/1000000000",
           "format": "time_series",
```

Comment:
i am thinking if we could filter out the gorli based chains from here since we cannot do much to maintain their balance 🤔

Reply:
it's already filtered because the `healthbot` does not produce those metrics (see the image in the PR description)

Reply:
ah! got it. it is filtered because wallet metrics are now coming from healthbot not API 👍